transcoder.go

```diff
@@ -118,7 +115,15 @@ func (t *Transcoder) registerService(svc *Service, svcOpts serviceOptions) error
 	}
 
 	if svcOpts.resolver == nil {
-		return fmt.Errorf("service %s is configured with a nil type resolver", svc.schema.FullName())
+		if canUseGlobalTypes(svc.schema) {
```

Comment:
nit: this can be moved into `resolverForService()`.

Reply:
done